google-cloud-firestore/pom.xml

```diff
@@ -157,6 +157,12 @@
       <version>3.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>proto-google-cloud-firestore-bundle-v1</artifactId>
+      <version>2.1.1-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Due to some overarching complexities in how our poms get built, bundled and released with the broader GCP libraries this declaration needs to change a little bit. I'll fix up all the pom files and push a commit to this branch with all the necessary changes, we can then review if you're interested in the details.